docs/standard/analyzers/platform-compat-analyzer.md

```diff
@@ -82,7 +133,7 @@ The analyzer does not check target framework moniker (TFM) target platforms from
     ```
 
   - **Inconsistent list**. If the lowest version for some platforms is `[SupportedOSPlatform]` while it is `[UnsupportedOSPlatform]` for other platforms, it's considered to be inconsistent, which is not supported for the analyzer. If inconsistency occurs, the analyzer ignores the `[UnsupportedOSPlatform]` platforms.
-  - If the lowest versions of `[SupportedOSPlatform]` and `[UnsupportedOSPlatform]` attributes are equal, the analyzer considers the platform as part of the **Supported only list**.
+    - If the lowest versions of `[SupportedOSPlatform]` and `[UnsupportedOSPlatform]` attributes are equal, the analyzer considers the platform as part of the **Supported only list**.
```

Comment:
Is this spacing correct?

Reply:
Yes, this is a sub scenario of "Inconsistent list"